x-pack/plugins/osquery/public/saved_queries/saved_query_flyout.tsx

```diff
@@ -51,6 +51,13 @@ const SavedQueryFlyoutComponent: React.FC<AddQueryFlyoutProps> = ({ defaultValue
     },
     [createSavedQueryMutation, onClose, serializer]
   );
+  const { euiTheme } = useEuiTheme();
+
+  // we need this flyout to be above the timeline flyout (which has a z-index of 1002)
+  const maskProps = useMemo(
+    () => ({ style: `z-index: ${(euiTheme.levels.flyout as number) + 4}` }),
```

Comment:
while I see why you went with `+ 4` here, I think sticking with `+ 3` is sufficient. If you decide to stick with `+ 4` I would update the comment and explain a bit better why... right now the comment says it has to be above the timeline with a 1002 z-index

Reply:
I'll try with +3, thanks!